include/fiction/algorithms/simulation/sidb/operational_domain.hpp

```diff
@@ -321,7 +307,7 @@ class operational_domain_impl
                           // for each y value in parallel
                           std::for_each(FICTION_EXECUTION_POLICY_PAR_UNSEQ y_indices.cbegin(), y_indices.cend(),
                                         [this, x](const auto y) {
-                                            is_operational({x, y});
+                                            is_sp_operational({x, y});
```

Comment:
Okay here, too. What is going on? Please search everywhere for this name, because there are a lot!

Reply:
@marcelwa This is correct here. The function which checks the operation status for a given ``sp``, is called 
``is_sp_operational({x, y});``

Reply:
Ah, I see. Could you then maybe rename it to `is_step_point_operational`?